Update BA connect cloud account with new role for consent the BA application (#2101)
EnterpriseDB/docs: product_docs/docs/biganimal/release/getting_started/02_connect_cloud_account.mdx

1. Create an Azure Active Directory Application client to delegate Identity and Access Management functions to Azure Active Directory (AD). You can create the Azure Active Directory Application using the Azure Portal, but a simpler and less error-prone approach is to use the `create-spn` script (see [Create Azure Active Directory Application using `create-spn`](#create-azure-active-directory-application-using-create-spn)). The script approach requires the Azure API.

!!! Note
    Some steps of the subscription process require approval of an Azure AD administrator.

>     Your Azure role must be the below either of the below roles in the Azure AD

Reviewer: ```suggestion
    Your Azure role in the Azure AD must be either:
```

      - Global Administrator
      - Privileged Role Administrator

>     or you need the cooperation of an user with the above roles in your organization.

Reviewer: ```suggestion
    or you need the cooperation of a user with the roles in your organization.
```

#### 4. Configure your account.
!!! Note

>     After step 1, you are prompted for approval by an Azure AD administrator. Your Azure AD's `Global Administrator` or `Privileged Role Administrator` can approve it.

Reviewer: ```suggestion
    After step 1, you are prompted for approval by an Azure AD administrator with either the Global Administrator or Privileged Role Administrator role.
```
